tests/treeTest.py

```diff
@@ -0,0 +1,19 @@
+from coalahtml.tree import Tree
+import unittest
+
+
+class TreeTest(unittest.TestCase):
+
+    def setUp(self):
+        self.FL = [
+                    "/home/tushar/coala-html/tests",
+                    "/home/tushar/coala-html/coalahtml",
+                    "/home/tushar/coala-html/tests/specs/directives/",
+                    "/home/tushar/coala-html/setup.py",
+                    "/home/tushar/coala-html/tests/TestUtilities.py"
+                  ]
+
+    def test_graph(self):
+        tr = Tree(self.FL)
+        testChild = tr.root.child['tests'].child.get('specs', None)
+        self.assertTrue(testChild)
```

Comment:
Actually, for creating better and working test scenario, I must obtain file lists by running coala. Presently, the file list has been hard coded and it won't work in travis or any other system - file paths are different. 
The next push has fixed this problem.